reinforcement_learning/examples/python/basic_usage.py

```diff
@@ -16,23 +16,31 @@ def main():
     model = rl_client.live_model(config, test_cb)
     model.init()
 
-    event_id = "event_id"
+    uuid = "event_id"
     context = '{"User":{"id":"a","major":"eng","hobby":"hiking"},"_multi":[{"N1":{"F1":"V1"},"N2":{"F2":"V2"}},{"N3":{"F1":"V3"}}]}'
 
-    model_id, chosen_action_id, actions_probabilities = model.choose_rank(event_id, context)
+    model_id, chosen_action_id, actions_probabilities = model.choose_rank(context, event_id=uuid)
 
-    print("event_id: " + event_id)
+    print("event_id: " + uuid)
     print("model_id: " + model_id)
     print("chosen action id: " + str(chosen_action_id))
     print("all action probabilities " + str(actions_probabilities))
 
-    model_id, chosen_action_id, actions_probabilities, event_id = model.choose_rank(context, deferred = True)
+    model_id, chosen_action_id, actions_probabilities, event_id = model.choose_rank(context, deferred = False)
+    model.report_action_taken(event_id)
+    print("event_id: " + event_id)
+    print("model_id: " + model_id)
+    print("chosen action id: " + str(chosen_action_id))
+    print("actions probabilities list: " + str(actions_probabilities))
+
+    model_id, chosen_action_id, actions_probabilities = model.choose_rank(context, deferred=True, event_id='asd')
```

Comment:
Maybe a more realistic event_id example is preferable?
Like for example: `e379df0b66fe48d1839e2a85a793c944y23fmJ`

Reply:
I agree with Yann and Tyler. I would do:
import uuid
uuid = str(uuid.uuid4())
model_id, chosen_action_id, actions_probabilities = model.choose_rank(context, event_id=uuid, deferred=True)